src/checkout-buttons/strategies/paypal-commerce/paypal-commerce-button-strategy.ts

```diff
@@ -57,6 +68,14 @@ export default class PaypalCommerceButtonStrategy implements CheckoutButtonStrat
         return Promise.resolve();
     }
 
+    private _onApproveHandler(data: ApproveDataOptions, actions: ApproveActions, cart: Cart) {
+        const { isHosted = true } = this._paymentMethod?.initializationData || {};
```

Comment:
This variable should not have default value. We should pass the value from backend

Reply:
removed, was hardcoded for test